js/script.js

```diff
@@ -30,8 +30,17 @@ const createForm = (taskInfo = {}) => {
 
 const createTaskElement = (task) => {
   const $listElement = document.createElement("li");
+  const $textContainer = document.createElement("span");
+  const $deleteButton = document.createElement("button");
   $listElement.setAttribute("id", task.id);
-  $listElement.innerText = task.title;
+  $deleteButton.setAttribute("id", `btn-${task.id}`);
```

Comment:
I think we don't need to declare id as we have direct button's reference and it is inside a function which creates individual reference for each button. Also you added `eventListener` directly to the button reference.